samples/net/aws_iot/README.rst

```diff
@@ -0,0 +1,255 @@
+   [00:00:08.736,572] <inf> aws_iot_sample: Connecting to AWS IoT
+   [00:00:08.736,633] <inf> aws_iot_sample: Next connection retry in 30 seconds
+   [00:00:08.736,663] <inf> aws_iot_sample: AWS_IOT_EVT_CONNECTING
+   [00:00:12.855,072] <inf> aws_iot_sample: AWS_IOT_EVT_CONNECTED
+   [00:00:12.856,323] <inf> aws_iot_sample: Publishing message: {"state":{"reported":{"uptime":12855,"app_version":"v1.0.0","modem_version":"nrf9160_1.3.4"}}} to AWS IoT shadow
+   [00:00:13.228,881] <inf> aws_iot_sample: AWS_IOT_EVT_READY
+   [00:00:13.244,781] <inf> aws_iot_sample: AWS_IOT_EVT_PUBACK, message ID: 32367
+   [00:01:12.867,675] <inf> aws_iot_sample: Publishing message: {"state":{"reported":{"uptime":72858,"app_version":"v1.0.0","modem_version":"nrf9160_1.3.4"}}} to AWS IoT shadow
+   [00:02:12.883,789] <inf> aws_iot_sample: Publishing message: {"state":{"reported":{"uptime":132874,"app_version":"v1.0.0","modem_version":"nrf9160_1.3.4"}}} to AWS IoT shadow
+
+.. note::
+   For nRF91 series boards the output will differ from the example output listed above.
+   This is because the sampe enables the :ref:`lib_at_host` library via the :kconfig:option:`CONFIG_AT_HOST_LIBRARY` option.
+   This library makes it possible to send and receive AT commands to and from the cellular modem using the nRF Connect for Desktop LTE Link monitor and Cellular Monitor applications.
+   The additional logs are AT command responses that the modem sends to the application core that is forwarded over UART to be picked up by any of the aforementioned Desktop applications.
+
+To observe incoming messages in the AWS IoT console, follow the steps documented in :ref:`aws_iot_testing_and_debugging`.
+
+Emulation
+=========
+
+The sample can be run in :ref:`QEMU x86 <zephyr:qemu_x86>` that simplifies development and testing and removes the need for hardware.
+Before you can build and run on QEMU, you need to perform the steps included in this link: :ref:`networking_with_qemu`.
+
+When the aforementioned steps are completed, you can build and run the sample by using the following commands:
+
+.. code-block:: console
+
+   west build -b qemu_x86 samples/net/mqtt
+   west build -t run
+
+Troubleshooting
+===============
+
+To enable more verbose logging from the AWS IoT library, set the :kconfig:option:`CONFIG_AWS_IOT_LOG_LEVEL_DBG` option.
+
+* If you are having issues with connectivity on nRF91 series boards, see the `Trace Collector`_ documentation to learn how to capture modem traces in order to debug network traffic in Wireshark.
+  The sample enables modem traces by default, set by the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE` option.
+* If you have issues with the sample, refer to :ref:`gs_testing`.
+* For issues related to connection towards AWS IoT, refer to :ref:`AWS IoT library troubleshooting <aws_iot_troubleshooting>`.
+
+Dependencies
+************
+
+This sample uses the following Zephyr libraries:
+
+* :ref:`mqtt_socket_interface`
```

Comment:
I'm a little unsure if we should list the libraries that are used "indirectly". Maybe MQTT and LTE LC can be dropped

Reply:
Agreed

Reply:
in this case: only

```
This sample uses the following Zephyr libraries:

* :ref:`net_if_interface`
* :ref:`lib_aws_iot`
* :ref:`net_mgmt_interface`
* :ref:`lib_hw_id`

```

persists really, but I think thats ok

Reply:
Yeah, looks good to me